svc-mesh/service-mesh.md

```diff
@@ -0,0 +1,229 @@
+# Kata Containers and service mesh for Kubernetes
+
+A service mesh is a powerful way to monitor and control the traffic
+between micro-services running in your Kubernetes cluster. They are
+powerful tools that you might want to use in combination with the
+security brought by Kata Containers.
+
+## How do they work ?
+
+They both rely on the same model, where they run controller applications
+in the control plane, and inject a proxy as a sidecar inside the service
+pod. The proxy registers to the control plane as a first step, and it
+constantly sends different sort of information about the service running
+inside the pod. Those information come from the filtering performed when
+receiving all the traffic initially intended for the service. That's how
+the interaction between the control plane and the proxy allows the user
+to apply load balancing and authentication rules to the on going traffic
+inside the cluster between multiple micro-services.
+
+I forgot to mention this magic could not happen without a good amount
```

Comment:
It gave me a good laugh but I'd drop, "I forgot to mention this magic".

Reply:
oh noooo.... I liked it !
Okay, I'll drop it...

Reply:
done